test/integration/self-hosted-notary/Dockerfile.populate_notary

```diff
@@ -0,0 +1,17 @@
+FROM debian:bullseye@sha256:ab2b95aa8d7d6d54866b92c322cf0693933c1ae8038652f24ddfda1d1763a45a
+
+COPY notary-service-container/ test/integration/self-hosted-notary/notary-service-container/
+
+RUN apt update &&\
+    apt upgrade -y && \
+    apt install -y docker notary expect ca-certificates && \
+    cp ./test/integration/self-hosted-notary/notary-service-container/server/ca.crt /usr/local/share/ca-certificates/notary_root_ca.crt && \
+    update-ca-certificates
+
+RUN chmod +x test/integration/self-hosted-notary/notary-service-container/notary_init.sh && \
+    chmod +x test/integration/self-hosted-notary/notary-service-container/notary_addhash.sh
```

Comment:
This should be part of the repo files, not the Dockerfile, I feel. Similar (probably same files) as for CI